app/assets/stylesheets/submissions.scss

```diff
@@ -56,65 +57,52 @@ ul.tag_list {
   list-style: none;
   margin: 0px;
   cursor: pointer;
-}
-
-ul.tag_list li {
-  margin-left: -40px;
-  position: left;
-
-  /* Transitional Elements. Browser compatibility provided */
-  -webkit-transition: all 0.2s;
-  -moz-transition: all 0.2s;
-  -ms-transition: all 0.2s;
-  -o-transition: all 0.2s;
-  transition: all 0.2s;
-}
-
-ul.tag_list li:hover {
-  color: $dark-blue;
-}
-
-ul.tag_list li ul {
-  text-align: left;
-  list-style: none;
-  margin: 0px;
-  cursor: pointer;
 
-  visibility: hidden;
-  opacity: 0;
-  position: absolute;
-
-  background-color: $light-grey;
-
-  /* The transition from hidden to opaque */
-  -webkit-transition: all 0.2s;
-  -moz-transition: all 0.2s;
-  -ms-transition: all 0.2s;
-  -o-transition: all 0.2s;
-  transition: all 0.2s;
-}
-
-ul.tag_list li ul li{
-  text-align: left;
-  padding: 2px;
-  color: $dark-blue !important;
-}
-
-ul.tag_list li ul li:hover{
-  background-color: $dark-grey;
-}
-
-ul li:hover ul {
-  /* When hover is activated on li, shows the menu */
-  display: block;
-  opacity: 1;
-  visibility: visible;
-}
-
-li.tag_list_search:hover, li.tag_list_title:hover {
-  /* Removes CSS formatting for titles and search bars. */
-  background-color: $light-grey !important;
-  cursor: default !important;
+  li {
+    margin-left: -40px;
+    position: left;
+    transition: all 0.2s;
+
+    &:hover {
+      color: $dark-blue;
+
+      ul {
+        /* When hover is activated on li, shows the menu */
+        display: block;
+        opacity: 1;
+        visibility: visible;
+      }
+    }
+
+    &.tag_list_search:hover,
+    &.tag_list_title:hover {
+      /* Removes CSS formatting for titles and search bars. */
+      background-color: $light-grey !important;
+      cursor: default !important;
+    }
+
+    ul {
+      background-color: $light-grey;
+      cursor: pointer;
+      list-style: none;
+      margin: 0;
+      opacity: 0;
+      position: absolute;
+      text-align: left;
+      transition: all 0.2s;
+      visibility: hidden;
+
+      li {
+        color: $dark-blue !important;
+        padding: 2px;
+        text-align: left;
+
+        &:hover{
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space